core/src/game/state.ts

```diff
@@ -32,7 +32,7 @@ export const initGameState = (): GameState => ({
 export const checkNoMoveLeftVictory = (board: Board): boolean =>
   getPlayableTilesForNextMove(board, getLastPlayedTile(board)).length === 0;
 
-export const checkIfGameWon = (gameState: GameState, board: Board): boolean => {
+export const checkIfGameWon = (board: Board): boolean => {
```

Comment:
Not blocking PR for this, but it would be great to actually have all win conditions checked inside this, by adding calls to getOppositePath.

Reply:
I agree